DOL.WHD.Section14c.Web/src/modules/routes.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+module.exports = function (app) {
+  
+  const ROUTE_PUBLIC = 1;
+  const ROUTE_LOGGEDIN = 3;
+  const ROUTE_USER = 7;
+  const ROUTE_ADMIN = 11;
+
+  let checkRouteAccess = function(route, userAccess) {
+    if (!route || !route.access) {
+      return true;
+    }
```

Comment:
This `checkRouteAccess` function and the `ROUTE_*` constants are still defined in `app.js`.  Rather than maintain them in two places, consider exporting them in `routes.js` and then using them in `app.js`.

Reply:
This was an oversight on my part. Forgot to remove it from app.js

Reply:
Those constants and the `checkRouteAccess` function are still referenced in `app.js` - that's why I suggested exporting them from `routes.js`.  😬   If you've got an alternative idea, though, go for it!